base_geoengine/static/src/js/widgets/geoengine_widgets.js

```diff
@@ -86,7 +88,7 @@ odoo.define('base_geoengine.geoengine_widgets', function (require) {
 
         _createLayers: function (field_infos) {
             this.vectorLayer = this._createVectorLayer();
-            this.rasterLayers = this.createBackgroundLayers([
+            this.rasterLayers = this.bgLayers.create([
                 field_infos.edit_raster,
             ]);
             this.rasterLayers[0].isBaseLayer = true;
```

Comment:
When we fallback on default (due to user error on raster layer definition) this.rasterLayers will be an empty list and fail.